docs/core/preview3/tools/using-ci-with-cli.md

```diff
@@ -78,8 +82,7 @@ for more information.
 
 ### AppVeyor
 
-The [appveyor.com ci](https://www.appveyor.com/) has .NET Core SDK preview2 already installed 
-in the build worker image `Visual Studio 2015`.
+The [appveyor.com ci](https://www.appveyor.com/) has .NET Core SDK preview2 already installed in the build worker image `Visual Studio 2015`.
```

Comment:
@blackdwarf do we need to create an issue to update these scripts for VS 2017? this topic still talks about vs 2015 when it's the preview 4 version